.circleci/publish-helm-charts

```diff
@@ -46,6 +46,16 @@ mkdir "${tmp_remote_repository}/${remote_subdirectory}/rhacs/${version}"
 cp -a "${central_services_chart}/rhacs" "${tmp_remote_repository}/${remote_subdirectory}/rhacs/${version}/central-services"
 cp -a "${secured_cluster_services_chart}/rhacs" "${tmp_remote_repository}/${remote_subdirectory}/rhacs/${version}/secured-cluster-services"
 
+mkdir -p "${tmp_remote_repository}/${remote_subdirectory}/opensource"
+
+echo "Packaging Helm chart for file ${central_services_chart}/opensource/Chart.yaml"
+helm package -d "${tmp_remote_repository}/${remote_subdirectory}/opensource" "${central_services_chart}/opensource"
+echo "Packaging Helm chart for file ${secured_cluster_services_chart}/opensource/Chart.yaml"
+helm package -d "${tmp_remote_repository}/${remote_subdirectory}/opensource" "${secured_cluster_services_chart}/opensource"
+
+echo "Building OSS helm repo index"
+helm repo index "${tmp_remote_repository}/${remote_subdirectory}/opensource"
```

Comment:
As for testing code in this file, have you tried running it? Does it successfully execute these new commands?
You can comment-out `git push` and `curl` commands below and run this script to see if produced changes in the repo are what you expect.

Reply:
The commands here are the same ones I ran manually to create the helm charts I pushed to helm-charts/opensource, but I'll run the test you suggested as well to double check.